website/content/docs/custom-widgets.md

```diff
@@ -1,16 +1,20 @@
 ---
 title: Creating Custom Widgets
-weight: 35
 group: fields
+weight: 35
 ---
+There are two ways to build a custom widget:
```

Comment:
This seems a bit off as it comes before the introduction:
> The NetlifyCMS exposes a `window.CMS` global...

I think we can drop it and the same way we have `Writing React Components inline`, we could have another section `Writing custom widgets as a separate package`

WDTY?

Reply:
Here is what I understood, am I right?
- Remove this part completely:
<img width="542" alt="Screen Shot 2020-08-13 at 15 06 26" src="https://user-images.githubusercontent.com/1583507/90138179-e40a9680-dd76-11ea-9dd0-23ed9bf893ed.png">
- Change this title from "Creating a custom widget using JSX syntax with a build step" to "Writing custom widgets as a separate package".
But I think it will make the document more readable if we mention at the beginning that there are two ways to create custom widgets.
What do you think?

Reply:
Yes, correct remove that section completely (like it was in the original version).

The title change is the one I'm mentioning in https://github.com/netlify/netlify-cms/pull/4144#discussion_r469923970

> But I think it will make the document more readable if we mention at the beginning that there are two ways to create custom widgets. What do you think?

Good point. How about we finish the detailed parts first and then if the doc gets too long we can section it using headers (e..g "Introduction", 'Writing Inline", "Separate Package"...)?

Reply:
it makes sense